eyre/src/context.rs

```diff
@@ -78,6 +82,7 @@ where
     }
 }
 
+#[cfg(feature = "anyhow")]
 impl<T> ContextCompat<T> for Option<T> {
```

Comment:
Maybe this could use `crate::ContextCompat` directly to reduce the number of `cfg`, as they are oftentimes harder to read in imports

Reply:
Fixed in https://github.com/eyre-rs/eyre/commit/3237c3895ebbc29ef83e47a7b49cbfb34205b2ee